### PR TITLE
Latest Changes on my End

### PR DESCRIPTION
**Pull request by KDRefugee**
Still Confused on this github.. Okay latest changes today was added multiple new vendors around and added them to preload, changed respawn times, added the new vendors to creatures, added new items to itemtemplate and updated other changes I had done

### CODE REVIEW

**Review by krssrb — APPROVED** [review]

**Review by krssrb — APPROVED** [review]

File: src/Rasa.DBL/Services/Preloader/SpawnpoolPreloader.cs [hunk 1]
```diff
@@ -47,7 +47,101 @@ protected override IEnumerable<object[]> GetRows()
             yield return new object[] { 30, 0, 0, 20, 810.12, 294.11, 376.28, 4.4, 1220, 30, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
             yield return new object[] { 100, 0, 0, 20, 870, 294.21, 385.5, 0.8, 1220, 100, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
             yield return new object[] { 101, 0, 0, 20, 870, 294.21, 388, 0.8, 1220, 101, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
-
-        }
+            yield return new object[] { 31, 0, 0, 20, -125.27734, 221.0586, -480.73047, 4.7, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+            yield return new object[] { 32, 0, 0, 20, -325.51172, 172.77344, -529.5, 0.96, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+            yield return new object[] { 33, 0, 0, 20, 159.10156, 163.07422, -95.57815, 1.53, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+            yield return new object[] { 34, 0, 0, 20, -700.16016, 170.13672, -337.96875, 5.61, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+            yield return new object[] { 35, 0, 0, 20, -652.65625, 283.64844, 876.22656, 2.9, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```
Comment (krssrb) [inline src/Rasa.DBL/Services/Preloader/SpawnpoolPreloader.cs]: You have only 25 values for spawnpool 31 - 36, there should be 27

File: src/Rasa.DBL/Services/Preloader/SpawnpoolPreloader.cs [hunk 2]
```diff
@@ -47,7 +47,101 @@ protected override IEnumerable<object[]> GetRows()
...
+            yield return new object[] { 36, 0, 0, 20, 752.45703, 293.96875, 397.08203, 4.7, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+			yield return new object[] { 36, 0, 0, 20, 752.45703, 293.96875, 397.08203, 4.7, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```
Comment (krssrb) [inline src/Rasa.DBL/Services/Preloader/SpawnpoolPreloader.cs]: duplicated entry with id 36

File: src/Rasa.DBL/Services/Preloader/SpawnpoolPreloader.cs [hunk 3]
```diff
@@ -47,7 +47,101 @@ protected override IEnumerable<object[]> GetRows()
...
+			yield return new object[] { 94, 0, 0, 25, 678.1328, 285.2539, 243.88281, 1.0, 1220, 44, 2, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+			yield return new object[] { 95, 0, 0, 25, 678.1328, 285.2539, 243.88281, 1.0, 1220, 44, 2, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+			yield return new object[] { 96, 0, 0, 20, 345.3047, 264.92188, 735.15234, 1.0, 1220, 58, 2, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+			yield return new object[] { 97, 0, 0, 20, 314.58594, 265.66406, 718.8368, 1.0, 1220, 58, 2, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+			yield return new object[] { 98, 0, 0, 20, 153.52734, 167.16406, 157.16016, 1.0, 1220, 59, 2, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+			yield return new object[] { 99, 0, 0, 20, 205.03906, 167.10938, 104.79297, 1.0, 1220, 59, 2, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+			yield return new object[] { 100, 0, 0, 20, 870.0, 294.21, 385.5, 0.8, 1220, 100, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+			yield return new object[] { 101, 0, 0, 20, 870.0, 294.21, 388.0, 0.8, 1220, 101, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```
Comment (krssrb) [inline src/Rasa.DBL/Services/Preloader/SpawnpoolPreloader.cs]: duplicated id for entries 100 and 101, those id's are on lines 47 & 48

**Review by krssrb — CHANGES_REQUESTED** [review]

File: src/Rasa.DBL/Services/Preloader/TeleporterPreloader.cs [hunk 4]
```diff
@@ -528,53 +528,56 @@ protected override IEnumerable<object[]> GetRows()
...
+            yield return new object[] { 614, 0, 0, "Waypoint: OMEGA_NOC_WP Dupe 468", 0.0, 0.0, 0.0, 0.0, 0 };
+            yield return new object[] { 615, 0, 0, "Waypoint: OMEGA_REEDUCATION_WP dupe 469", 0.0, 0.0, 0.0, 0.0, 0 };
+            yield return new object[] { 616, 0, 0, "Waypoint: OMEGA_MISSILE_CONTROL_WP dupe 470", 0.0, 0.0, 0.0, 0.0, 0 };
+            yield return new object[] { 617, 0, 0, "Waypoint: OMEGA_HOLDING_AREA_WP dupe 471", 0.0, 0.0, 0.0, 0.0, 0 };
+            yield return new object[] { 618, 0, 0, "Waypoint: OMEGA_PHARMCO_WP dupe 472", 0.0, 0.0, 0.0, 0.0, 0 };
+            yield return new object[] { 619, 0, 0, "Waypoint: OMEGA_BIOWEAP_WP dupe 473", 0.0, 0.0, 0.0, 0.0, 0 };
+            yield return new object[] { 620, 0, 0, "Waypoint: OMEGA_ICW_WP dupe 474", 0.0, 0.0, 0.0, 0.0, 0 };
+            yield return new object[] { 621, 0, 0, "Waypoint: OMEGA_PORTABLE_WP Dupe 467", 0.0, 0.0, 0.0, 0.0, 0 };
+            yield return new object[] { 622, 0, 2, "Waypoint: Tampeii Settlement", 408.89062, 542.28125, -114.91406, 0.0, 2048 };
```
Comment (krssrb) [inline src/Rasa.DBL/Services/Preloader/TeleporterPreloader.cs]: non exiting mapId 2048 at line 578

**Comment by jerrylf1971** [comment]
Hope to see all the new changes you guys have made 🙂

File: src/Rasa.DBL/Services/Preloader/TeleporterPreloader.cs [hunk 5]
```diff
@@ -298,76 +298,76 @@ protected override IEnumerable<object[]> GetRows()
...
+            yield return new object[] { 397, 0, 5, "Field Hospital: Timora Mines", -128.79803, 205.6074, 517.56011, 0.0, 1348 };
+            yield return new object[] { 398, 0, 5, "Hospital: Level 01: Entrance", 165.8984, -130.39257, 124.41682, 0.0, 1830 };
+            yield return new object[] { 399, 0, 5, "Hospital: Level 02: Barricade", -12.6972, -90.41950, 69.6952, 0.0, 1830 };
             yield return new object[] { 400, 25408, 3, "Wormhole: Torden Abyss", 170.28516, 542.0078, -558.0625, 2.53, 2028 };
-            yield return new object[] { 401, 0, 4, "Dropship Transport: Fort Virgil", -335.16016, 737.47266, -618.41016, 0.0, 2047 };
+            yield return new object[] { 401, 0, 0, "Dropship Transport: Fort Virgil See 266 Dupe", 0.0, 0.0, 0.0, 0.0, 0 };
             yield return new object[] { 402, 0, 4, "Dropship Transport: Quicksilver Post", -568.6211, 236.27734, -200.41406, 0.0, 1759 };
-            yield return new object[] { 403, 0, 0, "Baylor Base Dropship", 0.0, 0.0, 0.0, 0.0, 0 };
+            yield return new object[] { 403, 0, 0, "Dropship: Baylor Base", -594.95703, 258.22266, -517.10547, 0.0, 1759 };
```
Comment (krssrb) [inline src/Rasa.DBL/Services/Preloader/TeleporterPreloader.cs]: 403, change type from 0 to 4

**Review by krssrb — CHANGES_REQUESTED** [review]

**Pull request merged**